### DISCUSSION TITLE
how find the one ?

### DISCUSSION THREAD

**Discussion by ghost** [azerothcore/azerothcore-wotlk#13875] [Guides & Tips]
WaypointMovementGenerator::LoadPath: creature Eye of Dar'Khan (GUID Full: 0xf130003fc00041b3 Type: Creature Entry: 16320  Low: 16819) doesn't have waypoint path id: 0

the number of  creature  entry=16320   is many.  which is error.   how use  GUID Full ?

**Comment by virgo77** [comment]
Hi!
it's just a bug but it doesn't have effect about the spwn of this creature.
It's localized in Ghostland, in Deatholme and have many spawn points. I got the same on my server, and the creature is here and moving. Note that this creature does not have any specific path and doesn't particularly move around.

About the server, you will found 3 more errors like this. As the core is always fixed, if these ones are not fixed, it's because it is not really urgent to correct them :)

See you ^^

V.
